Setup gradle releases to maven central

germanattanasio opened the pull request:
Fixes #3

* Add support for releases to oss.sonatype
* Update Travis to deploy snapshot releases
* Update `README.md` and `RELEASES.md` file with information on how to release a new version

germanattanasio committed:
:new: setup gradle releases to maven central

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/watson-developer-cloud/spring-boot-starter?pullRequest=5) <br/>All committers have signed the CLA.

germanattanasio committed:
Rename jar to include watson fixes #4

:green_heart: Testing deployment

:green_heart: Second try

:green_heart: Switch to use tags

Auto release

mkistler reviewed (approved):
Looks good! 👍
